src/specsynthase/specbuilder.py

```diff
@@ -49,3 +50,7 @@ def add_spec(self, file_name):
 
     def dump(self, *args, **kwargs):
         return yaml.safe_dump(dict(**self), *args, **kwargs)
+
+    def validate(self):
+        validate_spec(self)
```

Comment:
Researching to fix the https://github.com/mozilla/balrog/pull/360, I read a little of the new spec. 
#8 will start with learning the new spec, because maybe https://swagger.io/specification/#referenceObject will kill spec-synthase :(